shared/teams/team/container.js

```diff
@@ -179,18 +182,19 @@ const mergeProps = (stateProps, dispatchProps, ownProps) => {
 
   const you = stateProps.you
   let youExplicitAdmin = false
-  let youImplicitAdmin = false
+  let youImplicitAdmin = stateProps.yourOperations.manageMembers
   let youAreMember = false
   if (you) {
-    youExplicitAdmin = Constants.isOwner(stateProps.yourRole) || Constants.isAdmin(stateProps.yourRole)
-    youImplicitAdmin = stateProps._implicitAdminUsernames.has(you)
+    // TODO: can we just test stateProps.yourOperations.RenameChannel ?
+    youExplicitAdmin = Constants.isOwner(stateProps.yourRole) || stateProps.yourOperations.manageMembers
```

Comment:
unclear if we need to check isOwner still?
Also, should there be a membership flag in userCanPerform?